File: code/player.py
class Player():
    def __init__(self):
        self.player1 = None
        self.player2 = None

    def choose_player(self):
        while True:
            player = input("¿Quieres ser X u O?").upper()
        
            if player == "X":
                self.player1 = "X"
                self.player2 = "O"
                print(f"El primer jugador es {self.player1} y el segundo jugador es {self.player2}.")
                break
            elif player == "O":
                self.player1 = "O"
                self.player2 = "X"
                print(f"El primer jugador es {self.player1} y el segundo jugador es {self.player2}.")
                break
            else:
                print("Elige X u O.")
            
        return self.player1, self.player2
    

File: code/js/player.js
class Player {
    constructor() {
        this.player1 = null;
        this.player2 = null;
    };

    choose_player() {
        while (true) {
            let player = prompt("¿Quieres ser X u O?").toUpperCase();

            if (player == "X") {
                this.player1 = "X";
                this.player2 = "O";
                console.log(`El primer jugador es ${this.player1} y el segundo jugador es ${this.player2}.`);
                break;
            } else if (player == "O") {
                this.player1 = "O";
                this.player2 = "X";
                console.log(`El primer jugador es ${this.player1} y el segundo jugador es ${this.player2}.`);
                break;
            } else {
                console.log("Elige X u O.")
            };
        };
        return [this.player1, this.player2];
    };
};
